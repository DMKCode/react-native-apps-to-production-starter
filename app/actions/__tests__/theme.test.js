import { changePrimaryColor } from '../theme';

it('creates a CHANGE_PRIMARY_COLOR action', () => {
  expect(changePrimaryColor('red')).toMatchSnapshot();
});
