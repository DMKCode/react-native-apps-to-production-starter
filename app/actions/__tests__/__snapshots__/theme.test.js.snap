// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a CHANGE_PRIMARY_COLOR action 1`] = `
Object {
  "color": "red",
  "type": "CHANGE_PRIMARY_COLOR",
}
`;
